crates/store/src/db/migrations/001-init.sql

```diff
@@ -145,3 +149,4 @@ CREATE TABLE
 
 CREATE INDEX idx_transactions_account_id ON transactions(account_id);
 CREATE INDEX idx_transactions_block_num ON transactions(block_num);
+CREATE INDEX unconsumed_network_notes ON notes(execution_mode, consumed);
```

Comment:
I believe this is correct; however we should check the query plan at some stage to see if its using this index at all.

Reply:
I've checked and its using this index.